assets/js/modules/analytics-4/datastore/api.js

```diff
@@ -38,8 +38,11 @@ const baseSelectors = {
 	 * @return {(boolean|undefined)} True, false, or undefined.
 	 */
 	isAdminAPIWorking( state ) {
-		// NOTE - This is against normal conventions. Usually we build selectors for everything
-		// we are working with an Alpha, unstable API at the time of development.
+		// This is against normal state access conventions.
+		// Usually we build selectors for any state access and avoid
+		// looking around state internals.
+		// But we are working with an Alpha, unstable Google API at the time
+		// of development.
```

Comment:
Nice @tofumatt !  Mine looks like a rant on Twitter in comparison

It makes me want to re-read The Elements of Style now I'm working on OSS :smile: